components/infra-proxy-service/storage/postgres/orgs.go

```diff
@@ -84,13 +105,18 @@ func (p *postgres) DeleteOrg(ctx context.Context, orgID uuid.UUID) (storage.Org,
 func (p *postgres) EditOrg(ctx context.Context, org storage.Org) (storage.Org, error) {
 	var o storage.Org
 
+	// ensure we do not pass null projects to db
+	if org.Projects == nil {
+		org.Projects = []string{constants.UnassignedProjectID}
```

Comment:
Is there something unusual in this service that suggests it should start with `UnassignedProjectID` ?
Prior art suggests this should be:
```suggestion
		org.Projects = []string{}
```

Reply:
Yes, it would be an empty assignment. but for now, I intentionally added it, in order to make it optional to bypass `CHECK (projects <> '{}')` so that currently implemented UI won't break.

let me know if it sounds weird?

Reply:
Being an "empty assignment" should not be a problem; other services do that. I guess I am still not understanding why this service should be treating it differently.

Reply:
yeah, that's current let's make it similar so that so it won't be confusing. so created issue for updating existing UI https://github.com/chef/automate/issues/3097